server/statistics/hot_regions_stat.go

```diff
@@ -13,9 +13,25 @@
 
 package statistics
 
+import (
+	"time"
+)
+
 // HotPeersStat records all hot regions statistics
 type HotPeersStat struct {
-	TotalLoads []float64     `json:"total_loads"`
-	Count      int           `json:"regions_count"`
-	Stats      []HotPeerStat `json:"statistics"`
+	TotalLoads     []float64         `json:"-"`
```

Comment:
What does it used for?

Reply:
update metircs, such as https://github.com/tikv/pd/blob/ab8b5b39930bde93c1258fbcfbc5d8b00eb64e40/server/cluster/coordinator.go#L514